tests/main/confinement-classic/hello-classic/Makefile

```diff
@@ -0,0 +1,77 @@
+SNAP_NAME = hello-classic
```

Comment:
Even though by now it is going to be built during the test it would be nice to begin using the naming convention here and call the snap `test-snapd-hello-classic`